src/shared/telemetry/telemetryUtils.ts

```diff
@@ -63,10 +63,6 @@ export function registerCommand<T>({
                 })
             }
 
-            if (result && result.datum) {
```

Comment:
If we tested that a DebugSession started and ended in the tests, would this change be necessary?

Reply:
yes, because on local run it is not